docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+<xode-block title="Java">
+
+```java
+public CompletableFuture createContinuous(final String projectionName, final String query)
+public CompletableFuture createContinuous(final String projectionName, final String query, CreateContinuousProjectionOptions options)
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+createContinuousProjection(projectionName: string, query: string, options?: CreateContinuousProjectionOptions): Promise<void>
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Update a projection
+
+Updates a projection. The name parameter is the name of the projection to be updated. The query parameter contains the new JavaScript.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Update
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Updating a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Update_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List all projections
+
+Returns a list of all projections.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListAll
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```
+// This is currently not available in the NodeJS client
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List continuous projections
+
+Returns a list of all continuous projections
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListContinuous
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+listContinuousProjections(options?: ListProjectionsOptions): Promise<ProjectionDetails[]>;
```

Comment:
Similar comment as https://github.com/EventStore/documentation/pull/404/files#r689297932 regarding the API Reference vs snippets.

Reply:
same as aboce commens , this needs a decision